client/src/app/site/mediafiles/services/media-manage.service.ts

```diff
@@ -0,0 +1,113 @@
+import { Injectable } from '@angular/core';
+import { HttpService } from 'app/core/services/http.service';
+import { ConfigService } from 'app/core/services/config.service';
+import { Observable } from 'rxjs';
+import { ViewMediafile } from '../models/view-mediafile';
+
+/**
+ * The structure of an image config object
+ */
+export interface ImageConfigObject {
+    display_name: string
+    key: string;
+    path: string;
+}
+
+/**
+ * The structure of a font config
+ */
+export interface FontConfigObject {
+    display_name: string
+    default: string;
+    path: string;
+}
+
+/**
+ * Holds the required structure of the manage payload
+ */
+interface ManagementPayload {
+    id: number,
+    key?: string,
+    default?: string,
+    value: ImageConfigObject | FontConfigObject
+}
+
+/**
+ * The service to manage Mediafiles.
+ *
+ * Declaring images as logos (web, projector, pdf, ...) is handles here.
+ */
+@Injectable({
+    providedIn: 'root',
+})
+export class MediaManageService {
+    /**
+     * Constructor for the MediaManage service
+     *
+     * @param httpService OpenSlides own HttpService
+     */
+    public constructor(private config: ConfigService, private httpService: HttpService) {}
+
+    /**
+     * Sets the given Mediafile to using the given management option
+     * i.e: setting another projector logo
+     *
+     * TODO: Feels overly complicated. However, the server seems to requires a strictly shaped payload
```

Comment:
If the server-side seems to handle things overcomplicated, try to slim it down a bit. The responsible corners of the code shouldn't be too hard to find.

Reply:
Not sure if altering the server would be required. If so, I would not know where to start.
Perhaps there are more elegant way's to create the payload that the server demands.